### PR TITLE
Remove real wfm name as examples

### PR DESCRIPTION
**Pull request by bishalgautam-ms**
Renamed WFM real name and GUID ID values.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

File: teams/teams-ps/teams/Get-CsTeamsShiftsConnectionInstance.md [hunk 1]
```diff
@@ -34,11 +34,11 @@ PS C:\> Get-CsTeamsShiftsConnectionInstance
...
-user@contoso.com    b75bbfa7-e92b-40fb-99a7-2d23b3404712 {Shift, SwapRequest, OpenShift, OpenShiftRequest…}     {Shift, SwapRequest, OpenShift, TimeOff…}                      "23005736-0000-0400-0000-60d10a990000" WCI-922cb572-369d-42d2-969a-d4927cd3683b   Test connector instance-4					10                 dfd24b34-ccb0-47e1-bdb…
-user@contoso.com    cbec0b9a-6161-4f6a-9067-9ec3eb881e56 {Shift, SwapRequest, UserShiftPreferences, OpenShift…} {Shift, SwapRequest, UserShiftPreferences, OpenShift…}         "2300589a-0000-0400-0000-60d132270000" WCI-da788d04-1bd2-48b1-a261-1dd80bde86db   Test connector instance-5					10                 dfd24b34-ccb0-47e1-bdb…
+user@contoso.com    12345fa7-e92b-40fb-99a7-2d23b3404712 {Shift, SwapRequest, UserShiftPreferences, OpenShift…} {Shift, TimeOffRequest}                                        "1d004f54-0000-0400-0000-60ce37120000" WCI-01c84e58-9a03-4e56-82f1-6b224132cad8   Test connector instance-1					10                 dfd24b34-ccb0-47e1-bdb…
```
Comment (Anitha-Tallam) [inline teams/teams-ps/teams/Get-CsTeamsShiftsConnectionInstance.md:37]: As diff admins create different instances, please use user1@contoso.com, user2@contoso.com...etc

File: teams/teams-ps/teams/Get-CsTeamsShiftsConnectionWfmTeam.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ Id      Name
 1000125 1000
 1000126 4500
 1000128 0006 - Microsoft 1
```
Comment (Anitha-Tallam) [inline teams/teams-ps/teams/Get-CsTeamsShiftsConnectionWfmTeam.md:51]: you can replace with WfmTeam1

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by bishalgautam-ms** [comment]
If someone has merge permission, Please merge this branch !!

**Pull request merged**